packages/core/src/components/Tooltip/index.tsx

```diff
@@ -95,6 +101,9 @@ export class Tooltip extends React.Component<TooltipProps & WithStylesProps, Too
 
   rafHandle: number = 0;
 
+  // Only used when Popover is enabled
+  delayTimer: number | null = null;
```

Comment:
you can simplify this logic by setting `delayTimeoutId = 0;` because they are always non-zero. then you don't have to do checks. (this is also consistent with how I initialized `rafHandle` above)